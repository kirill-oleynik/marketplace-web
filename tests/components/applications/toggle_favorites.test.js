const React = require('react');
const { shallow } = require('enzyme');
const toJSON = require('enzyme-to-json').default;
const ToggleFavorite = require('../../../app/components/applications/toggle_favorite').default;

describe('#render', () => {
  test('it renders correctly', () => {
    const component = shallow(
      <ToggleFavorite
        disabled
        addToFavorites={() => {}}
        removeFromFavorites={() => {}}
        application={{
          id: 1,
          favorite: {
            id: 1
          }
        }}
      />
    );

    expect(
      toJSON(component)
    ).toMatchSnapshot();
  });
});
