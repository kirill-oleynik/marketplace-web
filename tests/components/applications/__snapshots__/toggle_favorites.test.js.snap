// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#render it renders correctly 1`] = `
<Translate(RemoveFromFavorites)
  disabled={true}
  onClick={[Function]}
/>
`;
